astropy/utils/modal_utils.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+__all__ = ['make_modal']
+
+try:
+    import tkinter
+    from tkinter import ttk
+except ImportError:
+    tkinter = None
+
+if tkinter is None:
+    def make_modal(text, title, blink_ms, geometry, font, center_window=True):
+        print(text)
+else:
+    def make_modal(text, title, blink_ms, geometry, font, center_window=True):
```

Comment:
```suggestion
    def make_modal(text='CALCULATIONS\INCORRECT',
                   title='Are you sitting in front of a computer?',
                   blink_ms=750, geometry='550x155', font=('Helvetica', 48), color='r',
                   center_window=True):
```
https://github.com/astropy/nothing-to-see-here/issues/7#issuecomment-432301710 made a very pertinent point about sensible defaults.